keyboards/mt980/mt980.c

```diff
@@ -0,0 +1,35 @@
+#include "mt980.h"
+
+void keyboard_pre_init_user(void) {
+  /* Set NUMLOCK indicator pin as output */
+  setPinOutput(C6);
+  /* Set CAPSLOCK indicator pin as output */
+  setPinOutput(C7);
+  /* Set SCROLLOCK indicator pin as output */
+  setPinOutput(B5);
+}
+
+void led_set_user(uint8_t usb_led) {
+
+  if (usb_led & (1 << USB_LED_NUM_LOCK)) {
```

Comment:
You can use this instead

```suggestion
  if (IS_LED_ON(usb_led, USB_LED_NUM_LOCK)) {
```

(IS_LED_ON(usb_led, USB_LED_CAPS_LOCK))